webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -329,6 +329,27 @@ export class SakaiRubric extends RubricsElement {
       sakaiItemDelete.requestUpdate("rubric", this.rubric);
     }
   }
+
+ openEditWithKeyboard(e){
```

Comment:
Can you format this like the rest of the file, please? Also, you're checking for the same keycode twice.